menas/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
Logging for Menas will be configured separately (#154). I don't think using the spark default configuration from `logj4.properties.template` is appropriate for our webapp.